### ISSUE TITLE
Animation from planet view

### ISSUE THREAD

**Issue by webmazz** [mpetroff/pannellum#1172]
Is it possible to make an animation like in the example - zooming in from the "planet view"?
Example:  [CLICK](https://mapyinwestycji.pl/deweloperzy/tbv/osiedleeuropejskie/)

**Comment by mpetroff** [comment]
This is a duplicate of #1001, which is gated by stereographic projection support, tracked by #52.

The current answer is no.
